test/unit_tests/modules/util/atomic-helpers-spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var assert = chai.assert;
```

Comment:
Could we stick with expect/BDD. There should be a way to use .throw with the expect syntax.

http://chaijs.com/api/bdd/#method_throw